### ISSUE TITLE
Add dataset of CurvedLanes training?

### ISSUE THREAD

**Issue by jinfagang** [koyeongmin/PINet_new#14]
Is this dataset able to work with PINet?

https://github.com/xbjxh/CurveLanes

**Comment by koyeongmin** [comment]
Yes, but the contributor for the dataset is @pandamax . If you want to detail information about code for the dataset, It would be better to ask @pandamax .

**Comment by jinfagang** [comment]
@koyeongmin will it consider add this dataset into benchmark?

**Comment by jinfagang** [comment]
@koyeongmin  I notice that you have supported CurvedLane, but the result seems not as good as Tusimple:
![image](https://user-images.githubusercontent.com/21303438/97282936-3d646b00-187a-11eb-9d08-a301500a49b6.png)

![image](https://user-images.githubusercontent.com/21303438/97282976-47866980-187a-11eb-9ca0-7afe11c740fb.png)
![image](https://user-images.githubusercontent.com/21303438/97283003-4d7c4a80-187a-11eb-8faa-2a4f20d439ae.png)


Do u have any idea?

**Comment by koyeongmin** [comment]
I have not tried to train CurvedLane yet. So I don't know the detail. I recommend training more and fixing some hyperparameters because your pictures are similar when the model is not trained sufficiently on CULane dataset.

**Comment by jinfagang** [comment]
@koyeongmin You mean you not train CurvedLane dataset? I am just test your code: https://github.com/koyeongmin/PINet_new/tree/master/CurveLanes

Does this model not trained on curvedlanes?

**Comment by koyeongmin** [comment]
Codes for CurveLanes is contributed by @pandamax . I don't know exactly how long or how to train the dataset.  Also, Curvelanes looks more difficult than TuSimple and CULane, so performance will be lower than them.
